src/api/docker-compose.yml

```diff
@@ -0,0 +1,108 @@
+################################################################################
```

Comment:
It looks like you're adding this file back again.  NOTE: we've moved it to `docker/docker-compose.yml`, which is where you should add the user service definition.

Reply:
Yeah, not sure what happened here. Strange rebase issue that must have gone over my head as I didn't create this file myself.